packages/rsdd/rsdd.0.1.0/opam

```diff
@@ -0,0 +1,45 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Bindings for RSDD"
+description:
+  "OCaml bindings for the Rust Decision Diagram (RSDD) library, generated with ocaml-rs."
+maintainer: ["Matt Wang <mxw@cs.washington.edu>"]
+authors: [
+  "Steven Holtzen <s.holtzen@northeastern.edu>"
+  "Matt Wang <mxw@cs.washington.edu>"
+]
+license: "MIT"
+homepage: "https://github.com/neuppl/rsdd-ocaml"
+bug-reports: "https://github.com/neuppl/rsdd-ocaml/issues"
+depends: [
+  "ocaml" {>= "4.08.0" & < "5"}
+  "dune" {>= "2.7" & >= "3.0"}
```

Comment:
can be simply `{>= "2.7"}`. this is likely due to having a version constraint on [this line](https://github.com/neuppl/rsdd-ocaml/blob/9bb86e392e04e8455f8d6088efa3f1999ee11830/dune-project#L18C1-L18C1)

 If you are depending on any features / fixes that are only available on version 3.0 or later, it's probably a better idea to have `(lang dune 3.0)` on [this line](https://github.com/neuppl/rsdd-ocaml/blob/9bb86e392e04e8455f8d6088efa3f1999ee11830/dune-project#L1)

Reply:
Appreciate it (and I don't think we're depending on dune 3); this was autogenerated earlier and I hadn't realized that there was a conflict.

Reply:
Thanks for taking actions! Just for you info: https://github.com/neuppl/rsdd-ocaml/commit/fc2965dac9a63c061445a3e6e493bcac9e3e053d#r129203968